linked_list/LinkedList.java

```diff
@@ -0,0 +1,217 @@
+/*
+* Following is the code for Linked List
+* T defines the type of Linked List to be created
+*/
+
+import java.util.NoSuchElementException;
+
+// Class to create a Node that contains data and pointer to next Node 
+class Node<T> {
+    T data;     // Data of type T
+    Node<T> next;     // Pointer to next Node
+
+    // Constructor to create a Node
+    public Node(T input) {  
+        this.data = input;
+        this.next = null;
+    }
+}
+
+// Class containing methods to be performed on Linked List
+class LinkedList_<T> {
```

Comment:
@Monal5031 I think @singhpratyush is talking about only making the main method `static`, which it usually is. 
All other methods will be object methods.